dist/api_data.py

```diff
@@ -71,16 +70,13 @@ def __ge__(self, other):
         this option is no longer supported, retained for backward compatibility''',
         type='list', choices=['write_timestamp'], undoc=True),
     Config('write_timestamp_usage', 'none', r'''
-        describe how timestamps are expected to be used on table modifications. This option should
-        be used in conjunction with the corresponding \c write_timestamp configuration under the
-        \c assert option to provide errors and assertions for incorrect timestamp usage. The
-        choices are the default, which ensures that once timestamps are used for a key, they are
-        always used, and also that multiple updates to a key never use decreasing timestamps,
-        \c mixed_mode, which additionally allows updates with no timestamp even after timestamps
-        are first used, and \c never which enforces that timestamps are never used for a table.
-        (The \c always, \c key_consistent and \c ordered choices should not be used, and are
-        retained for backward compatibility.)''',
-        choices=['always', 'key_consistent', 'mixed_mode', 'never', 'none', 'ordered']),
+        describe how timestamps are expected to be used on table modifications.  The choices are
+        the default, which ensures that once timestamps are used for a key, they are always used,
+        and also that multiple updates to a key never use decreasing timestamps and \c never which
+        enforces that timestamps are never used for a table.  (The \c always, \c key_consistent,
+        \c mixed_mode and \c ordered choices should not be used, and are retained for backward
+        compatibility.)''', choices=['always', 'key_consistent', 'mixed_mode', 'never', 'none',
```

Comment:
The mixed mode option here is for backward compatibility?

Reply:
Yes, MDB is still calling into the WT build with `mixed_mode` set.